backend/api-service/src/api_inventory_service.go

```diff
@@ -174,6 +174,15 @@ func (s *InventoryApiService) ListLocations(w http.ResponseWriter) error {
 
 // NewInventoryTransaction - Create a new Inventory Transaction
 func (s *InventoryApiService) NewInventoryTransaction(inventoryTransaction InventoryTransaction, w http.ResponseWriter) error {
+	if inventoryTransaction.Action == "" {
+		return EncodeJSONStatus(http.StatusBadRequest, "Empty required field: action", w)
+	}
+	if inventoryTransaction.ItemId == "" {
+		return EncodeJSONStatus(http.StatusBadRequest, "Empty required field: item_id", w)
+	}
+	if inventoryTransaction.LocationId == "" {
+		return EncodeJSONStatus(http.StatusBadRequest, "Empty required field: location_id", w)
```

Comment:
this could be a helper function given how many times it's needed in this file.

Reply:
Done.